public/app/core/components/Breadcrumbs/Breadcrumb.tsx

```diff
@@ -0,0 +1,69 @@
+import { css, cx } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Icon, LinkButton, useStyles2 } from '@grafana/ui';
+
+import { Breadcrumb as BreadcrumbType } from './types';
+
+type Props = BreadcrumbType & {
+  isCurrent: boolean;
+};
+
+export function Breadcrumb(props: Props) {
```

Comment:
Personal preference 😃 

When having a list of items component, and an item component, to avoid mistakes missing a letter when singular/plural I like to call them `List`, `ListItem`. I have seen this practice in other components like `TagItem`, `SearchItem` or `NavBarItem`. What do you think about calling this `BreadcrumbItem`

Reply:
great suggestion! avoids the naming clash with the type as well 😄